### DISCUSSION TITLE
load test

### DISCUSSION THREAD

**Discussion by camrun91** [ApollosProject/apollos-templates#405] [Q&A] [answered]
We are getting closer to going live with our app. However we are wanting to load test it as we are seeing some slowness. When I look at the documentation of the loadtests it looks like it is not set up to load test the new tab feeds and such yet. just wanted to make sure this is on the radar.

**Comment by redreceipt** [comment, marked as answer]
> We are getting closer to going live with our app. However we are wanting to load test it as we are seeing some slowness. When I look at the documentation of the loadtests it looks like it is not set up to load test the new tab feeds and such yet. just wanted to make sure this is on the radar. 

If you're looking for a simple load testing solution before there is a robust core alternative, checkout wrk or artillery. Both are scriptable and can be used to testing any query you want.

**Comment by redreceipt** [comment]
@camrun91 is this resolved?

**Reply by camrun91** [reply]
Yes it is
